src/services/color/manipulation.ts

```diff
@@ -41,6 +43,26 @@ export const shade = (color: string, ratio: number) => {
   return inOriginalFormat(color, shade);
 };
 
+/**
+ * Returns the tinted color for light mode and shaded color for dark mode
+ * @param color - Color to mix with white
+ * @param ratio - Mix weight. From 0-1. Larger value indicates more white.
+ * @param colorMode - Light or dark only
+ */
+export const tintOrShade = (color: string, ratio: number, colorMode: any) => {
```

Comment:
Not sure we can use for the `colorMode` type the following:

```suggestion
export const tintOrShade = (color: string, ratio: number, colorMode: UseEuiTheme['colorMode']) => {
```